VOKMockUrlProtocol.m

```diff
@@ -23,7 +23,7 @@
     #endif
 #endif
 
-static NSString *const MockDataDirectory = @"VOKMockData";
+static NSString *const DefaultMockDataDirectory = @"VOKMockData";
```

Comment:
I made it so the property will return the default if set to nil, which is kind of in the same vein. Would you prefer I do this too?